### ISSUE TITLE
[REQ] Ability to search (via keyword) for settings, options

### ISSUE THREAD

**Issue by laurenweinstein1** [cagnulein/qdomyos-zwift#1371]
QZ is so incredibly comprehensive with so many settings and options -- and is updated so quickly -- that I'm frequently noticing users posting bugs that were already fixed but via settings that they don't know about it. This just happened to me! Since the documentation (quite understandably!) can be out of date, it might be very helpful if there were some sort of way to search for all settings/options that applied to a given keyword (multiple word strings would be even better, but even single keywords would be good). No need (though it would be nice) to be able to go right to the found settings/options, just a keyword (not case sensitive, of course) match list would be great.

So, if a user is having problems with Strava, they could just do a search on "strava" and get a list of all related settings/options, and go from there. 

I realize that this is a relatively radical suggestion, but it could be increasingly useful (and time saving, especially for new users) going forward. Thanks!

**Comment by cagnulein** [comment]
yes like for example in the android or ios settings, where you can search a keyword. Yes I thought about this, as always it's only a matter of time :)
